main/tests/server/src/com/google/refine/tests/expr/functions/CrossFunctionTests.java

```diff
@@ -9,6 +9,7 @@
 import java.util.List;
 import java.util.Properties;
 
```

Comment:
Thanks for updating the existing tests! It would be great if we could also have one or two tests to demonstrate the functionality that you have added.